### DISCUSSION TITLE
Different results between `dotnet test` and `xunit.console.exe`

### DISCUSSION THREAD

**Discussion by cskwrd** [xunit/xunit#2471] [Question] [answered]
I am trying to test a solution for a legacy code base. It is a mix of C# and VB.NET targeting .NET 4.8. The code under test use the old project file format and the old `packages.conf` NuGet package format. The unit test projects are all C# and use the new formats.

I have followed the steps outlined in the [docs](https://xunit.net/docs/getting-started/netfx/cmdline) for testing .NET from the command line.

When I execute the tests using the command `dotnet test -f net48 BusinessLogic.Tests.dll`, all tests pass as expected. Using the command `xunit.console.exe BusinessLogic.Tests.dll`, most of the tests fail. It seems that assembly references are resolved differently between the tools. Is that expected behavior?

**Comment by bradwilson** [comment, marked as answer]
No, those two things should generally give the same results.

Can you provide a repro project?

**Reply by bradwilson** [reply]
Please open an issue with a repro project. Thanks!
